frontend/pages/hosts/details/MacSettingsModal/MacSettingsModal.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Button from "components/buttons/Button";
+import Modal from "components/Modal";
+import { IMacSettings } from "interfaces/mdm";
+import MacSettingsTable from "./MacSettingsTable";
+
+interface IMacSettingsModalProps {
+  hostMacSettings: IMacSettings; // TODO: define this type when API shape is determined
+  isLoading: boolean;
+  onClose: () => void;
+}
+
+const baseClass = "mac-settings-modal";
+
+const MacSettingsModal = ({
+  hostMacSettings,
+  isLoading,
+  onClose,
+}: IMacSettingsModalProps) => {
+  return (
+    <Modal title="macOS settings" onExit={onClose} className={baseClass}>
+      <>
+        <MacSettingsTable
+          isLoading={isLoading}
+          hostMacSettings={hostMacSettings}
+        />
+        <div className="modal-cta-wrap">
+          <Button type="submit" variant="brand" onClick={onClose}>
```

Comment:
I'm not sure if we want to use `type="submit"` when there is no form being submitted?